benchmarks/fetch/headers-length32.mjs

```diff
@@ -19,7 +19,7 @@ const headers = new Headers(
     'Width',
     'Accept-CH',
     'Via',
-    'Refresh',
```

Comment:
why was this removed?

Reply:
This is because I replaced it with set-cookie.
The length, which is a fast-path condition, must be 32.